crates/crypto/.gitignore

```diff
@@ -0,0 +1 @@
+/Cargo.lock
```

Comment:
Not necessary? After you added it to the workspace, it will not create any Cargo.lock in this directory

Reply:
fixed